.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+<!--- Provide a general summary of the issue in the Title above -->
```

Comment:
HTML comments do not get rendered in markdown on GitHub.  Is this your intent?  Typically, when providing a template for opening issues I recommend placing the full text in a code block.  This makes it easy for a user to copy and paste the intended template.